Alignment/HIPAlignmentAlgorithm/scripts/configure.pl

```diff
@@ -2,17 +2,13 @@
 use File::Basename;
```

Comment:
I don't think there is any need to change the name of configure.pl, which is a perl script. Besides, this script is not something new. I do not mean that we will not make any further improvements on this, but it is beyond the scope of this PR.